starter-code/ShoppingListVer2/app/src/main/java/ly/generalassemb/drewmahrt/shoppinglistver2/DbHelper.java

```diff
@@ -0,0 +1,29 @@
+package ly.generalassemb.drewmahrt.shoppinglistver2;
+
+import android.content.Context;
+import android.database.sqlite.SQLiteDatabase;
+import android.database.sqlite.SQLiteOpenHelper;
+
+/**
+ * Created by ds on 10/25/16.
+ */
+
+public class DbHelper extends SQLiteOpenHelper {
+
+    public static final int DATABASE_VERSION = 1;
+    public static final String DATABASE_NAME = "grocery.db";
+
+    public DbHelper(Context context) {
+        super(context, DATABASE_NAME, null, DATABASE_VERSION);
+    }
+
+    @Override
+    public void onCreate(SQLiteDatabase db) {
+
```

Comment:
In this method we need code to create a new table in the database. The column names, table name, and the SQL statement to create the table should be written above as constants.